packages/viz/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "experimentalDecorators": true,
     "lib": [
       "dom",
-      "es2015"
+      "es2015",
+      "es2019"
```

Comment:
Should we use just ESNext? 
Or if we need es2019, shouldn't we remove es2015?

Reply:
We cannot use ESNext because we wouldn't have this line converted then:
https://github.com/CartoDB/toolkit/blob/9ad98437694f8580d4a75e6a82cf07094fca94e5/packages/viz/src/lib/interactivity/viewport-features/aggregations.ts#L30

But that said, I think that I would use es2019 and then transpile code with Babel.